src/elements/selection-expansion-panel/selection-expansion-panel.ts

```diff
@@ -169,9 +182,22 @@ export class SbbSelectionExpansionPanelElement extends SbbHydrationMixin(LitElem
 
     this._checked = input.checked;
     this._disabled = input.disabled;
+    this._sizeAttributeObserver.observe(input, { attributeFilter: ['size'] });
     this._updateState();
   }
 
+  private _onSizeAttributesChange(mutationsList: MutationRecord[]): void {
+    for (const mutation of mutationsList) {
+      if (mutation.attributeName === 'size') {
+        const group = this._group;
+        const size = (mutation.target as HTMLElement).getAttribute('size')!;
+        if (!group || group.size === size) {
```

Comment:
just for my understanding, is there a reason for setting the data size when group.size === size ?

Reply:
second question, if the group has a size set, how is it coming down to the selection expansion panel? If I understand it correctly, only the mutation of the input (panel) is observed.

Reply:
you're right :) i'll add a note in the code

about your first comment, potentially a user could use the group with an array of selection-expansion with inner panels, and then set the size of a single inner panel; in that case i would assume nothing would change in the selection-expansion 
![image](https://github.com/user-attachments/assets/5d1d3902-0af1-4661-8ad9-e5029cd34c39)

Reply:
Would probably make sense to also document this, or test this case (maybe it's already, I did not check ;) )

Reply:
documentation already there :)